script.js

```diff
@@ -0,0 +1,16 @@
+const menuBars = document.querySelector('.menu-bars');
+const close = document.querySelector('#close');
+const nav = document.querySelector('#desktop-nav');
+// const logo = document.querySelector('#logo');
```

Comment:
- [ ] Please remove the commented code because it is against the JavaScript best practices. Also, it makes your code look not clean.